composer.json

```diff
@@ -44,7 +44,7 @@
         "drupal/piwik": "~1.0",
         "drupal/piwik_reporting_api": "dev-1.x",
         "drupal/r4032login": "dev-1.x#391f10d69b9b8c466a2fddac2a6b99f1289e7781",
-        "drupal/rdf_entity": "^1.0.0-alpha5",
+        "drupal/rdf_entity": "dev-draft_for_taxonomies",
```

Comment:
Now that we are not using the 8.x-1.x version in our code, until we have a new alpha, we should not any more require another branch in composer, because even when the upstream patch is accepted, it will not get into the codebase. We will need to keep using the 1.0.0-alpha5 and use the PR as a patch.